src/modules/claims/claims.service.ts

```diff
@@ -1,17 +1,14 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { providers, utils, Wallet } from 'ethers';
+import { verifyCredential } from 'didkit-wasm-node';
```

Comment:
@jrhender @JGiter we will need to create separate services based on env. for claims service now, right? For 'didkit-wasm-node' and 'didkit-wasm' - like for VC Service?